extensions/jms/src/main/java/org/mule/extensions/jms/api/source/JmsListener.java

```diff
@@ -116,26 +112,36 @@
 
   @Parameter
   @Optional
+  @Example(EXAMPLE_CONTENT_TYPE)
   @Summary("The content type of the message body")
```

Comment:
all of these fields are missing jdocs. Summary is completementary to the Jdoc, but not a replacement of it. Otherwise the generated docs will suck